.prettierrc.json

```diff
@@ -3,5 +3,6 @@
     "tabWidth": 4,
     "trailingComma": "es5",
     "printWidth": 140,
-    "semi": true
+    "semi": true,
+    "arrowParens": "avoid"
```

Comment:
"At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well as making other changes. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the default value for the option."

always as default.

Varför använder du avoid här?

Reply:
Just cleaner in my opinion and I haven't had any concerns or problems with above. For me it's more consistent with other code bases. But it doesn't really matter for me which we choose. What's your opinion?